newton/_src/solvers/kamino/examples/sim/example_sim_test_all_joints.py

```
@@ -0,0 +1,357 @@
+
+
+@wp.kernel
+def _test_control_callback(
+    state_t: wp.array(dtype=float32),
+    control_tau_j: wp.array(dtype=float32),
+):
+    """
+    An example control callback kernel.
+    """
+    # Set world index
+    wid = int(0)
```

Ruff (0.14.8)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)